### ISSUE TITLE
Hypothesis testing explained in plain English

### ISSUE THREAD

**Issue by Dave-Schultz** [leios/SoME_Topics#226]
## About the author

I have been tutoring statistics for 15 years and have expertise across a wide variety of stats concepts. I have a passion for perfecting my craft, and have been improving this particular talk through hundred of lectures. 

## Quick Summary

Hypothesis testing is one of the fundamental concepts of statistics and is taught in every stats class, but very few students and not even many stats teachers really understand how it works.  I have created a video describing hypothesis testing in plain English for beginning and intermediate statistics students. It does a much clearer job of explaining hypothesis testing concepts to ordinary students than do the usual methods. 

## Target medium

I have an existing video (my first). I would like the result to be a video as well.

### More details

While (I believe) the content of my video is very helpful, the video I have made is entirely amateurish in production. It is my first video and I have no idea what I'm doing production-wise. A link to the video is below.

https://www.loom.com/share/624adfe7a68047f8b008b2ac85305c12

### Contact details

I can be contacted at cestmois@gmail.com

### Additional context

I am posting this for the Summer of Math Exposition for 3Blue1Brown.

(Any additional licensing information? If you do not say anything, this post will be considered [CC-BY](https://creativecommons.org/licenses/by/2.0/).)

**Comment by reduccionista** [comment]
Hi, this is a very central topic to statistics. What do you like to add to your video? maybe visuals or animations.

**Comment by Dave-Schultz** [comment]
Thanks for your reply!

As you say, this is a very central topic and is generally quite poorly
taught.

Really, what I want is to work with someone who knows what they are doing
and make a really nice version of this talk.
As you no doubt saw, the "production quality" of this video was pretty bad;
it's the first video I ever made and I was nervous and not used to giving a
lesson without a student to talk to. I would be happy to work with you to
make a new video done with professional quality.

Does that sound good/reasonable to you? Do you have other thoughts? I'd
love to hear where you are coming from.

Dave

**Comment by reduccionista** [comment]
I mainly make animations for videos, but it seems that you want collaboration in the whole production of the video, and I have to admit that is out of my reach, especially in the sound production.

**Comment by Dave-Schultz** [comment]
It looks like we're not a good match here. Thank you so much for your
offer, though!
